app/assets/stylesheets/common/communicarts.scss

```diff
@@ -1,6 +1,6 @@
 @import "constants";
 @import "webflow";
-@import "../communicart_mailer/email_status";
+@import "../view_specific/shared/email_status";
```

Comment:
The name "view_specific" feels a little awkward to me. Whats the reason you don't want to just call it "view"?